networking/configuring-network-operator.adoc

```diff
@@ -0,0 +1,21 @@
+[id="configuring-network-operator"]
+= Configuring networking
+include::modules/common-attributes.adoc[]
+:context: configuring-network-operator
+
+toc::[]
+
+The Network Operator installs and upgrades the networking components on an
+{product-title} cluster. If you want to customize the network configuration, you
+must modify the settings before installing the cluster.
+
+[IMPORTANT]
+====
+You cannot modify your network settings after you install the cluster.
+====
+
```

Comment:
I'd make installing the cluster by following the network configuration instructions an assembly prereq and link to it.

Reply:
Copied to Google Doc to visualize in context.